crates/ef-testing/src/models/case.rs

```diff
@@ -230,6 +230,9 @@ impl BlockchainTestCase {
         }
 
         if !diff.is_empty() {
+            if let Ok(revert_reason) = maybe_revert_reason {
+                diff.push(format!("revert reason: {}", revert_reason));
+            }
```

Comment:
can you rename `diff` to translate the fact that it contains an accumulation of the errors encountered in the post state?